bucket/ffsend.json

```
@@ -1,15 +1,16 @@
 {
+    "version": "0.2.54",
     "homepage": "https://github.com/timvisee/ffsend",
-    "description": "Easily and securely share files from the command line. A fully featured Firefox Send client.",
+    "description": "Easily and securely share files with Firefox Send.",
     "license": "GPL-3.0-only",
-    "version": "0.2.54",
+    "depends": "version/openssl-light",
```

https://github.com/ScoopInstaller/Versions
https://github.com/lukesampson/scoop/blob/master/buckets.json#L4

```suggestion
    "depends": "versions/openssl-light",
```